### DISCUSSION TITLE
PGBouncer and exporter update?

### DISCUSSION THREAD

**Discussion by evanasse** [apache/airflow#29110] [General]
We are using pgbouncer, pgbouncer-exporter and statsd-exporter images, and they contain a fair amount of vulnerabilities as they are years old now.
Is there a plan to update them soon, or should we try to rebuild them ourselves to fix the vulnerabilities?

**Comment by potiuk** [comment]
No plans but it's a good idea to update them. 

If you rebuild and test them yourself with some more recent version - you could make a PR to update versions in https://github.com/apache/airflow/tree/main/chart/dockerfiles is a good idea (and I can build and push the new images after merge). As of recently the statsd-exporter is just a "plain" statsd-exporter without any modification from our side so you can test it with newer versions and PR directly to the chart: https://github.com/apache/airflow/blob/main/chart/values.yaml#L82

Would be awesome if you could test it before we merge those changes.

**Reply by evanasse** [reply]
Related PR: https://github.com/apache/airflow/pull/29792
